apps/silverback-drupal/composer.lock

```diff
@@ -75,6 +75,36 @@
             },
             "time": "2021-04-13T14:13:44+00:00"
         },
+        {
+            "name": "amazeelabs/proxy-gutenberg",
+            "version": "1.0.0",
+            "dist": {
+                "type": "path",
+                "url": "../../packages/composer/amazeelabs/proxy-gutenberg",
```

Comment:
This reference is correct within this context, but on a separate project it will not. I assume that we'd just need to add a `repository` section into the `composer.json` file for this, right?

Reply:
This is only here. In another project it would just be loaded from packagist.

Reply:
Perfect.